### PR TITLE
Feature/wdp191000 16 rwd stopka

### PR DESCRIPTION
**Pull request by JagodaBodnar**
Problem: Footer not working in small resolutions.
I created new classes for smaller resolutions. I decided to center content of footer links on small screens and for that purpose also wrapped image (cards) in div to be able to center that element too. According to client request I also aligned copyrights -to the left for sm and md.
image
image

### CODE REVIEW

File: src/partials/90_footer.html [hunk 1]
```diff
@@ -45,19 +45,19 @@ <h6>Orders</h6>
...
       </div>
     </div>
   </div>
   <div class="bottom-bar">
     <div class="container">
       <div class="row align-items-center">
-        <div class="col"></div>
-        <div class="col copyright text-center">
-          <p>©Copyright 2016 Bazar | All Rights Reserved</p>
+        <div class="col-sm-12 col-md-12 col-lg-4"></div>
```
Comment (Neonides) [inline src/partials/90_footer.html]: `"col-sm-12 col-md-12` można zastąpić `col-sm-12`

File: src/sass/mixin.scss [hunk 2]
```diff
@@ -0,0 +1,38 @@
+$screen-sm: 576px;
+
+$screen-md: 768px;
+
+$screen-lg: 992px;
+
+$screen-xl: 1200px;
+
```
Comment (izabela-d) [inline src/sass/mixin.scss]: Wydaje mi się, że zmienne dotyczące szerokości ekranu powinny być w _variables.scss

File: src/partials/90_footer.html (hunk 1 above)
Comment (JagodaBodnar) [reply src/partials/90_footer.html]: changed

File: src/sass/mixin.scss (hunk 2 above)
Comment (JagodaBodnar) [reply src/sass/mixin.scss]: zmienione, ale nie moge wypushowac

File: src/sass/components/_footer.scss [hunk 3]
```diff
@@ -4,16 +4,40 @@ footer {
     padding: 3rem 0;
 
     .menu-wrapper {
+      &-image {
+        @include respond-to(md) {
+          display: flex;
```
[the remark this answers is not in the document]
Comment (JagodaBodnar) [reply src/sass/components/_footer.scss]: done

**Pull request merged**